PrebidMobile/AdUnits/MultiformatAdUnit/BidInfo.swift

```diff
@@ -0,0 +1,35 @@
+/*   Copyright 2019-2023 Prebid.org, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import Foundation
+
+@objcMembers
+@objc(PBMBidInfo)
+public class BidInfo: NSObject {
+  
+    public private(set) var result: ResultCode
```

Comment:
Please rename it to `resultCode`. The word result is to abstract for this property.